includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -476,10 +484,23 @@ class="location-input wc-enhanced-select dropdown"
 				<?php esc_html_e( 'I will also be selling products or services in person.', 'woocommerce' ); ?>
 			</label>
 
-			<?php if ( 'unknown' === get_option( 'woocommerce_allow_tracking', 'unknown' ) ) : ?>
+			<?php
+			if ( 'unknown' === get_option( 'woocommerce_allow_tracking', 'unknown' ) ) {
+				$tracking_opt_out = true;
+
+				// EU should be opt-in.
+				if ( in_array( $country, WC()->countries->get_european_union_countries(), true ) ) {
+					$tracking_opt_out = false;
+				}
+
+				// Respect the DNT header.
+				if ( ! empty( $_SERVER['HTTP_DNT'] ) ) { // WPCS: input var ok.
+					$tracking_opt_out = false;
+				}
+				?>
 				<div class="woocommerce-tracker">
 					<p class="checkbox">
-						<input type="checkbox" id="wc_tracker_optin" name="wc_tracker_optin" value="yes" checked />
+						<input type="checkbox" id="wc_tracker_optin" name="wc_tracker_optin" value="yes" <?php $tracking_opt_out ? 'checked' : ''; ?> />
```

Comment:
`<?php $tracking_opt_out ? 'checked' : ''; ?>`

This doesn't ever actually echo the word 'checked'. It just evaluates an expression and does nothing with it. :P

Should probably use [`checked( $tracking_opt_out )`](https://codex.wordpress.org/Function_Reference/checked) anyways since it's clearer.

Reply:
Accident, promise ;p